xpdacq/xpdacq.py

```diff
@@ -257,6 +257,48 @@ def calibration(sample, scanplan, auto_dark = glbl.auto_dark, **kwargs):
     _execute_scans(scan, auto_dark, auto_calibration = False, light_frame = True, dryrun = False)
     return
 
+def background(sample, scanplan, auto_dark = glbl.auto_dark, **kwargs):
```

Comment:
Check if it is better to do it this way or to do ``auto_dark = None`` then inside the function
```
if auto_dark == None:
    auto_dark = glbl.auto_dark
```
I don't mind which way it is, but I thought that @tacaswell  liked it better this way....